DEPLOYMENT_GUIDE.md

```
@@ -0,0 +1,191 @@
+```bash
+cd d:\Downloads\Personal_Projects\BizPilot2
+git add .
+git commit -m "feat: Add DigitalOcean App Platform deployment config"
+git push origin main
+```
+
+---
+
+## Step 2: Create DigitalOcean App
+
+1. Go to **https://cloud.digitalocean.com/apps**
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)